bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -8,13 +8,15 @@
 		<item-type>Rollershutter</item-type>
 		<label>Position</label>
 		<description>The vertical position of the shade</description>
-		<category>blinds</category>
+		<category>Blinds</category>
+		<state min="0" max="100" pattern="%.1f %%"/>
```

Comment:
Update instructions are needed for the corresponding channel definition to be updated for managed things. See https://next.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types